src/ui/simulator/windows/inspector/property.update.cpp

```diff
@@ -924,28 +924,29 @@ bool InspectorGrid::onPropertyChanging_ThermalCluster(wxPGProperty*,
     if (name == "cluster.gen-ts")
     {
         long index = value.GetLong();
-
-        Data::LocalTSGenerationBehavior behavior
-          = Data::LocalTSGenerationBehavior::useGlobalParameter;
+        using Behavior = Data::LocalTSGenerationBehavior;
+        auto behavior = Behavior::useGlobalParameter;
```

Comment:
Maybe **behavior** should be renamed to be more expressive, even if it's a local variable in a local context

Reply:
It' clear from lines 925 and 928 that this a TS generation behavior.